.env.production

```diff
@@ -1,5 +1,8 @@
 NEXT_PUBLIC_ALGOLIA_APP_ID=XQEP3AD9ZT
 NEXT_PUBLIC_ALGOLIA_API_KEY=118b51a67eefbe4d3c445d1e1090afac
 NEXT_PUBLIC_SHOPIFY_DOMAIN=ifixit-test.myshopify.com
-NEXT_PUBLIC_SHOPIFY_STOREFRONT_ACCESS_TOKEN=bc6c5d0972b3ad5d565c99768e6065bf
-NEXT_PUBLIC_IFIXIT_API_ORIGIN=https://www.cominor.com
\ No newline at end of file
+NEXT_PUBLIC_SHOPIFY_STOREFRONT_ACCESS_TOKEN=d08db995952e88a37804b54a284e5130
+NEXT_PUBLIC_IFIXIT_API_ORIGIN=https://www.cominor.com
+WP_ORIGIN=https://valkyrie.ifixit.com
+WP_BASIC_AUTH_USER=ifixit
+WP_BASIC_AUTH_PASSWORD=iFixitIsTheNews!
```

Comment:
We should avoid keeping secrets plaintext in repo.

Reply:
@djmetzle are these credentials protecting sensitive API? Since posts are public I thought these were like public access tokens.

Reply:
No, that is there to add an extra security guard over `wp-admin`.

That secret is public now.

Reply:
Oh I'm really sorry for that, I think these needs to be changed 😬 This would also mean that to deploy this on Shopify I would need a public endpoint without credentials, since we can only fetch from APIs client side on the Shopify theme @Annika29

Reply:
Have we rotated this secret @djmetzle ?

Reply:
No, i haven't. Code ref: https://github.com/iFixit/ifixit/blob/877045a67c5d316d10dce1ca4dcbdf4e0fc2664c/Conf/varnish/production.vcl#L583